test/spec/OTAgentSpec.js

```diff
@@ -39,8 +39,9 @@ describe("JSONPatchOTAgent instance", function () {
       applyPatch = jasmine.createSpy("applyPatch");
       var apply = function apply(){
         applyPatch.apply(this, arguments);
+        return arguments[0]; //obj
```

Comment:
Why `return applyPatch.apply(this, arguments);` is not enough?
Isn't `argument[0]` incorrect, in case of root-replacing patches?

Reply:
applyPatch is just a spy. It doesn't have any logic.